src/sentry/tasks/commit_context.py

```diff
@@ -155,76 +150,25 @@ def process_commit_context(
         with lock.acquire():
             metrics.incr("sentry.tasks.process_commit_context.start")
 
-            cache_key = DEBOUNCE_CACHE_KEY(group_id)
-
             set_current_event_project(project_id)
 
             project = Project.objects.get_from_cache(id=project_id)
             set_tag("organization.slug", project.organization.slug)
 
-            owners = GroupOwner.objects.filter(
-                group_id=group_id,
-                project=project,
-                organization_id=project.organization_id,
-                type=GroupOwnerType.SUSPECT_COMMIT.value,
-            )
             basic_logging_details = {
                 "event": event_id,
                 "group": group_id,
                 "organization": project.organization_id,
             }
-            # Delete old owners
-            to_be_deleted = owners.filter(
-                date_added__lte=django_timezone.now() - PREFERRED_GROUP_OWNER_AGE
-            )
-
-            if len(to_be_deleted):
-                for record in to_be_deleted:
-                    record.delete()
-
-            current_owners = owners.filter(
-                date_added__gte=django_timezone.now() - PREFERRED_GROUP_OWNER_AGE
-            ).order_by("-date_added")
-
-            if len(current_owners) >= PREFERRED_GROUP_OWNERS:
-                # When there exists a Suspect Committer, we want to debounce this task until that Suspect Committer hits the TTL of PREFERRED_GROUP_OWNER_AGE
-                cache_duration = django_timezone.now() - current_owners[0].date_added
-                cache_duration = (
-                    cache_duration
-                    if cache_duration < PREFERRED_GROUP_OWNER_AGE
-                    else PREFERRED_GROUP_OWNER_AGE
-                )
-                cache.set(cache_key, True, cache_duration.total_seconds())
-                metrics.incr(
-                    "sentry.tasks.process_commit_context.aborted",
-                    tags={
-                        "detail": "maxed_owners_none_old",
-                    },
-                )
-                logger.info(
-                    "process_commit_context.maxed_owners",
-                    extra={
-                        **basic_logging_details,
-                        "reason": "maxed_owners_none_old",
-                    },
-                )
-                return
 
             code_mappings = get_sorted_code_mapping_configs(project)
 
             frames = event_frames or []
-            munged = munged_filename_and_frames(event_platform, frames, "munged_filename", sdk_name)
```

Comment:
No need to call the munging function anymore since the new logic does it later on https://github.com/getsentry/sentry/pull/62768